Software/Scratch/GoPiGoScratch.py

```diff
@@ -359,32 +359,22 @@ def run(self):
 		elif msg.lower()=="IR".lower():
 			print "IR!"
 			if en_ir_sensor==0:
-				import lirc
-				try:
-					sockid = lirc.init("keyes", blocking = False)
-					en_ir_sensor=1
-				except:
-					en_ir_sensor=0
-					print("*******************************")
-					print("ERROR: IR Receiver not enabled!")
-					print("Please run Advanced Communications Option and enable it")
-					print("A REBOOT will be necessary")
-					print("*******************************")
+				import ir_receiver
+				en_ir_sensor=1
```

Comment:
we don;t need it. I just wanted to make minimal changes to it for now. There were too many other things in there too. On that note, I couldn;t figure out why there were 2 IR code blocks in there: https://github.com/karan259/GoPiGo-1/blob/a2c2690fd47d10bd8e0ccb7e792fa5d74cb82654/Software/Scratch/GoPiGoScratch.py#L402-L422 Are we still using this